### PR TITLE
Add page track additional properties and override page name in adobe provider

### PR DESCRIPTION
**Pull request by jptinsman**
- **What kind of change does this PR introduce?**
Adds additional properties to Page Track and also add ability within Adobe Omniture provider to override page name and not be forced to use the site URL.

- **What is the current behavior? Link to open issue?**
Not able to add additional parameters to page tracking https://github.com/angulartics/angulartics2/issues/351

- **What is the new behavior?**
page tracking allows passing of additional parameters and adobe can override page name from the url

Note, I am using Angular 5 and Angulartics2 v5.4.0 but those are only tags and not branches so it would be nice if the previous major release versions could have an updated branch so that they can have this feature to.

### CODE REVIEW

File: src/lib/providers/adobeanalytics/adobeanalytics.ts [hunk 1]
```diff
@@ -14,18 +14,27 @@ export class Angulartics2AdobeAnalytics {
...
+      .subscribe((x) => this.pageTrack(x.path, x.properties));
     this.angulartics2.eventTrack
       .pipe(this.angulartics2.filterDeveloperMode())
       .subscribe((x) => this.eventTrack(x.action, x.properties));
     this.angulartics2.setUserProperties
       .subscribe((x) => this.setUserProperties(x));
   }
 
-  pageTrack(path: string) {
+  pageTrack(path: string, properties: any) {
```
Comment (jptinsman) [inline src/lib/providers/adobeanalytics/adobeanalytics.ts]: what is the return type for both `pageTrack` and `eventTrack`? The original provider and all the other providers don't show any return types (neither `void` nor `any`).

Comment (scttcper) [reply src/lib/providers/adobeanalytics/adobeanalytics.ts]: i think its just an object of some kind

Comment (Bizotics-Dev) [reply src/lib/providers/adobeanalytics/adobeanalytics.ts:25]: Can we send here new property with category and label ? If yes then please guide me how we will send here .
Currently i send like below way but it not workout.It is not tracked in ga report.
this.angulartics2.eventTrack.next({
    action: action,
    properties: {
    category: category,
     label: label,
     search_query:'xyz'
}});

**Comment by jtinsman-mgage** [comment]
I haven't worked on Angular in years and even this change was just to make public a change I had made for my project. Why was this never merged and instead just closed? I still feel that it is a valuable feature to define properties on page track

**Pull request closed without merging**